### PR TITLE
Feature/prune dead code

### PR DESCRIPTION
**Pull request by abe-one**
# Description
Fixes # (issue)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status
- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?
- [x] Manually

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by francishtknguyen** [comment]
Looks great so far, getting rid of a lot of technical debt!

**Review by francishtknguyen — DISMISSED** [review]

File: api/allIncidents/adminIncidentsRouter.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ router.use(authRequired);
  * @swagger
  * /dashboard/incidents:
  *  get:
- *    summary: Path returning all pending incidents in reverse chronological order
+ *    Summary: Path returning all pending incidents in reverse chronological order
```
Comment (JustinPeczenij) [inline api/allIncidents/adminIncidentsRouter.js]: Don't think we want to revert this change!

File: api/app.js [hunk 2]
```diff
@@ -25,13 +21,10 @@ const swaggerUIOptions = {
 
 //###[  Routers ]###
 const indexRouter = require('./index/indexRouter');
-const profileRouter = require('./profile/profileRouter');
```
Comment (JustinPeczenij) [inline api/app.js:28]: Seems like Okta will handle the admin profiles

**Review by francishtknguyen — APPROVED** [review]

File: data/migrations/20210713225521_updated-table.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 exports.up = function (knex) {
   return knex.schema
     .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
-    .createTable('incidents', (incidents) => {
+    .createTable('force_ranks', (incidents) => {
```
Comment (JustinPeczenij) [inline data/migrations/20210713225521_updated-table.js:4]: Good catch

**Review by JustinPeczenij — APPROVED** [review]
Technical debt reduced successfully
